app/src/main/java/chat/rocket/android/app/AppLifecycleObserver.kt

```diff
@@ -31,14 +31,8 @@ class AppLifecycleObserver @Inject constructor(
 
     private fun changeTemporaryStatus(userStatus: UserStatus) {
         launch {
-            val currentServer = serverInteractor.get()
-            val account = currentServer?.let { getAccountInteractor.get(currentServer) }
-            val client = account?.let { factory.create(currentServer) }
-
-            try {
-                client?.setTemporaryStatus(userStatus)
-            } catch (exception: RocketChatException) {
-                Timber.e(exception)
+            serverInteractor.get()?.let { currentServer ->
```

Comment:
Great!

Reply:
@luciofm You could leave the `currentServer ->` since it is a simple code (`it` is explicitly here- even more by declarating `serverInteractor.get()`).